src/symbol/mod.rs

```diff
@@ -277,3 +280,132 @@ impl Dwarf {
         self.rent(|parsed| parsed.get_var_address(name))
     }
 }
+
+pub struct RelocatedDwarf(Vec<RelocatedDwarfEntry>);
```

Comment:
This struct probably can be moved into its own module, but can be done later.

Reply:
Will do this later.